src/hooks/useKeyboardButtonPressedFunc.js

```diff
@@ -6,7 +6,7 @@ const KEYDOWN_ENTER_EVENT = 13;
 export function useKeyboardButtonPressedFunc(onClick) {
   return useCallback(
     e => {
-      if (e.keyCode === KEYDOWN_SPACE_EVENT || e.keyCode === KEYDOWN_ENTER_EVENT) {
+      if ((onClick && e.keyCode === KEYDOWN_SPACE_EVENT) || e.keyCode === KEYDOWN_ENTER_EVENT) {
         onClick(e);
```

Comment:
something seems a bit off - if `onClick === undefined`, and `e.keyCode === KEYDOWN_ENTER_EVENT`, we will call `onClick(e)` even though `onClick` is undefined. Didn't TypeScript shout on this?
in any case, I think you can do the following -
```suggestion
      if ([KEYDOWN_ENTER_EVENT, KEYDOWN_SPACE_EVENT].includes(e.keyCode)) {
        onClick?.(e);
```